pkg/kubelet/kubelet.go

```diff
@@ -1975,6 +1970,10 @@ func (kl *Kubelet) handleMirrorPod(mirrorPod *v1.Pod, start time.Time) {
 // HandlePodAdditions is the callback in SyncHandler for pods being added from
 // a config source.
 func (kl *Kubelet) HandlePodAdditions(pods []*v1.Pod) {
+	// Responsible for checking limits in resolv.conf
```

Comment:
This probably should go into loop, because we need to check for each pod that is getting added.